source/_integrations/fritz.markdown

```diff
@@ -91,3 +93,55 @@ Parental control switches are designed for advanced users, thus they are disable
 
 Due to security reasons, AVM implemented the ability to enable/disable a port forward rule only from the host involved in the rule.
 As a result, this integration will create entities only for rules that have your Home Assistant host as a destination.
+
+**Note 1**: On your FRITZ!Box, enable the setting `Permit independent port sharing for this device` for the device which runs HA (`Internet` -> `Permit Access` -> `<Name of HA device>`)
```

Comment:
I don't have such settings enabled and port sharing works just fine

Reply:
Are you sure? It used to be nescessary in the old custom component

Reply:
Yes I am, but leave it it won't hurt